### PR TITLE
feat: Added single video page

### PR DESCRIPTION
**Pull request by deepak29-git**
Added single video
added like,dislike,watchlater and save to playlist icon
Screenshot (132)

### CODE REVIEW

**Review by Richa-rani02 — APPROVED** [review]
Suggestion: raise PR wrt to a functionality only ,i can see changes in files which is related to refactoring .better to do refactoring in different PR

**Comment by Richa-rani02** [comment]
Suggestions: Deploy preview link as well its difficult to visualize by seeing code what are the changes

**Review by ajay-mandaviya — APPROVED** [review]
everything looks good to me ,  
if there is a more than one or two file use index.js file in  for this folder and import  it into index and export from index so 
it's good if you also show the deploy link and

File: src/Pages/SingleVideoPage/SingleVideoPage.jsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import { useNavigate, useParams } from "react-router-dom";
+import React from "react";
+import ReactPlayer from "react-player/youtube";
+import "../SingleVideoPage/SingleVideoPage.css";
+import { Header } from "../../components/Header/Header";
+import { useEffect } from "react";
+import { getVideos } from "../../Data/getvideos";
+import { useLoader } from "../../Custom-hook/use-loader";
+import { useData } from "../../Context/data-context";
+import { useState } from "react";
```
Comment (Hardiegogo) [inline src/Pages/SingleVideoPage/SingleVideoPage.jsx:10]: useEffect and useState can be imported in a single statement.

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -1,30 +1,56 @@
...
+              Log out
+            </button>
+          ) : (
+            <button
+              onClick={() => navigate("/login")}
+              className="login-btn btn primary-bg"
+            >
+              Login
+            </button>
+          )}
```
Comment (Hardiegogo) [inline src/components/Header/Header.jsx:50]: empty spaces can be removed for better readability.

**Review by Hardiegogo — COMMENTED** [review]
I recommend adding the preview link, so that we can also check the working of the app.
Otherwise very modular code with a good project structure.

**Pull request merged**